label_sync/README.md

```diff
@@ -1,8 +0,0 @@
-# Label Sync
```

Comment:
Maybe it is better to have some sort of documentation like this to document changes in this PR since AFAIU we are moving from updating labels via **bazel** to **CronJob**? (PR description could be used for that which is already good source of information IMO)

Reply:
Done